python_modules/libraries/dagster-powerbi/dagster_powerbi/resource.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict
+
+import requests
```

Comment:
nit: we should use `httpx`, since it gives us the capability for async requests if we need it https://www.python-httpx.org/#features